docs/reference/lifecycle_callbacks.md

```diff
@@ -28,6 +28,8 @@ Method       | Invoked by Stimulus…
 initialize() | Once, when the controller is first instantiated
 connect()    | Anytime the controller is connected to the DOM
 [name]TargetConnected(target: Element) | Anytime a target is connected to the DOM
+attributeChanged(attributeName: string, oldValue: string | null, newValue: string | null) | Anytime the controller element's attributes change
```

Comment:
This needs corresponding prose elsewhere. It's also worth noting that Values are better suited for observing `data-` prefixed changes, and that `attributeChanged` is better for global attributes like `[disabled]`, `[hidden]`, or `aria-*` prefixed attributes.